features/support/publishing_cuke_helpers.rb

```diff
@@ -6,10 +6,12 @@ def write_in_publisher(txt)
   end
 
   def append_to_publisher(txt)
-    status_message_text = find("#status_message_text").value
-    fill_in id: "status_message_text", with: "#{status_message_text} #{txt}"
+    input =  find("#status_message_text")
```

Comment:
- Layout/SpaceAroundOperators: Operator `=` should be surrounded by a single space.
- Layout/ExtraSpacing: Unnecessary spacing detected.